src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/WindowsForms/MyAppFileAccessor.cs

```diff
@@ -13,8 +13,10 @@ internal class MyAppFileAccessor : IMyAppFileAccessor, IDisposable
     private readonly IServiceProvider _serviceProvider;
     private readonly UnconfiguredProject _project;
     private readonly IProjectThreadingService _threadingService;
+    private readonly IPhysicalProjectTreeStorage _storage;
     private DocData? _docData;
     private MyAppDocument? myAppDocument;
```

Comment:
I know it's not this PR, but if you're editing this file could you add an underscore to this field name so that it's consistent with our naming style?